### PR TITLE
Release/1.5.1

### PR DESCRIPTION
**Pull request by baptistedonaux**

### CODE REVIEW

**Comment by JohnMcLear** [comment]
Hey @baptistedonaux This is a good start but there are a bunch the CHANGELOG is missing.  We also merge the changes to the ether.github.com repo at the same time if you can take a look at the release procedure it should cover those bits.

We also require a tagged version.. 

We also require a version is available in release/* although I'm not sure how you'd do that :)

Good luck!

**Comment by baptistedonaux** [comment]
I added the CHANGELOG (I hope that it's good). I wait your return.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 1.5.1
+ * NEW: Export into HTML
```
Comment (JohnMcLear) [inline CHANGELOG.md]: How is this new?

**Comment by JohnMcLear** [comment]
Getting closer :) 

So we'd still need a release version on the ether/ repo

We'd also need docs ready to go.

Also the binaries need making.

Take a look through the docs again please..  

https://github.com/ether/etherpad-lite/wiki/Release-procedure

Thanks for this btw, having someone who can help doing releases will be massive for us if you can find the time!

**Comment by JohnMcLear** [comment]
@baptistedonaux you might want to pull again to version for release :)

**Comment by JohnMcLear** [comment]
@baptistedonaux any progress?

**Comment by baptistedonaux** [comment]
Others comments ?

**Comment by JohnMcLear** [comment]
@baptistedonaux please read https://github.com/ether/etherpad-lite/wiki/Release-procedure

**Comment by Gared** [comment]
Already released

**Pull request closed without merging**